test/e2e/storage/testsuites/volumes.go

```diff
@@ -68,12 +76,22 @@ func (t *volumesTestSuite) getTestSuiteInfo() TestSuiteInfo {
 }
 
 func (t *volumesTestSuite) skipUnsupportedTest(pattern testpatterns.TestPattern, driver drivers.TestDriver) {
+}
```

Comment:
was it intended to remove skip persistent test from `skipUnsupportedTest`? I would think you'd want to add all the various `skipXTest` functions into `skipUnsupportedTest` since it gets called "before resource initialization"